source/documentors/references/templates/doc_concept_template.rst

```diff
@@ -3,15 +3,15 @@ Concept Document Template
 
 A concept is a document that clarifies and illuminates a particular topic. It is understanding-oriented. It could be considered a conceptual guide. This type of document answers a why question. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context. 
 
-For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.”  (Replace this text with your content).
+For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions”.
```

Comment:
My comment about the image example was that this line ("What is an XBlock") might be clearer if the associated image _also_ was about the same thing. So you changed the image to one about Superset, which is still unrelated to this line about XBlocks. It's not a big deal but that was what I was saying.

Reply:
You are right. Sorry about the confusion.